lib/trento/application/integration/discovery/payloads/host_discovery_payload.ex

```diff
@@ -3,7 +3,16 @@ defmodule Trento.Integration.Discovery.HostDiscoveryPayload do
   Host discovery integration event payload
   """
 
-  @required_fields :all
+  @required_fields [
+    :hostname,
+    :ip_addresses,
+    :ssh_address,
+    :agent_version,
+    :cpu_count,
+    :total_memory_mb,
+    :socket_count,
+    :os_version
+  ]
```

Comment:
just curious, is this needed also we the default? isn't `:all` enough?

Reply:
The `installation_source` field is not mandatory, as old versions of the agent won't send it, and we need to cover this use case

Reply:
You are right, having a `default` value already fills the value, so `:all` works perfectly fine. I will revert it